<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>511</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>QFile</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEditFile"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>QTextStream</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEditTextStream">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>85</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>85</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled/>
         </palette>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpen_IODevice"/>
   <addaction name="actOpen_TextStream"/>
   <addaction name="separator"/>
   <addaction name="actSave_IODevice"/>
   <addaction name="actSave_TextStream"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actOpen_IODevice">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/804.bmp</normaloff>:/images/images/804.bmp</iconset>
   </property>
   <property name="text">
    <string>QFile直接打开</string>
   </property>
   <property name="toolTip">
    <string>用QFile直接打开文本文件</string>
   </property>
  </action>
  <action name="actOpen_TextStream">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/122.bmp</normaloff>:/images/images/122.bmp</iconset>
   </property>
   <property name="text">
    <string>QTextStream打开</string>
   </property>
   <property name="toolTip">
    <string>用QtextStream打开文本文件</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actSave_IODevice">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/104.bmp</normaloff>:/images/images/104.bmp</iconset>
   </property>
   <property name="text">
    <string>QFile另存</string>
   </property>
   <property name="toolTip">
    <string>用QFile直接另存文件</string>
   </property>
  </action>
  <action name="actSave_TextStream">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/066.GIF</normaloff>:/images/images/066.GIF</iconset>
   </property>
   <property name="text">
    <string>QTextStream另存</string>
   </property>
   <property name="toolTip">
    <string>用QTextStream另存文件</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
